src/renderer/renderer_elements.ts

```diff
@@ -854,6 +857,30 @@ export class InterstateEdge extends Edge {
             container.style.display = 'none';
     }
 
+    public drawLabel(
```

Comment:
This doesn't make space for the rest of the graph. Wouldn't this overlap other nodes? What does this look like?

Reply:
It _can_ in theory overlap with other nodes, as yes, there is no space reservation. However, the placement is at the start of the interstate edge (see image). So the overlap should only happen if:
a) the layout is bad to begin with (nodes basically touching), or
b) the interstate edge label is extremely long (e.g., very complex, huge condition)

![image](https://github.com/spcl/dace-webclient/assets/9193712/c407ba66-5413-4654-9f06-f66b99d09f9f)

Reply:
Ok. Thanks for the screenshot